src/python/WMCore/WebTools/RESTModel.py

```diff
@@ -170,6 +171,27 @@ def _addMethod(self, verb, methodKey, function, args=[],
             self.methods[verb] = {}
         @wraps(function)
         def wrapper(*input_args, **input_kwargs):
+            if dump_request_info:
+                request = cherrypy.request
+                headers = dict(request.headers)
+                skip = ['cookie', 'cms-authn-hmac', 'cms-authz-admin']
+                for key in headers.keys():
+                    if key.lower().startswith('ssl-'):
```

Comment:
How about
`if key.lower().startswith('ssl-') or key.lower() in skip:`
?

`skip` also sounds like we can just skip those keys, while you're actually removing them from the headers. Should we rename it to `keysToRemove` or something like that?

Reply:
done

Reply:
I guess you forgot to push your changes. I still see both if statements.

Reply:
Alan, please check now.